tasks/rh-subscription.yml

```diff
@@ -61,4 +61,4 @@
   command: "dnf module enable -y {{ ovirt_repositories_rh_dnf_modules | join(' ') }}"
   args:
     warn: False
-  when: ovirt_repositories_ovirt_version >= 4.4
+  when: ovirt_repositories_ovirt_version|string >= '4.4'
```

Comment:
Another solution can be `(ovirt_repositories_ovirt_version|string).split('-')[0]|float >= 4.4` but will have same result